src/sidebar/components/selection-tabs.js

```diff
@@ -30,26 +29,28 @@ function Tab({
   const title = count > 0 ? `${label} (${count} available)` : label;
 
   return (
-    <button
-      className={classnames('selection-tabs__type', {
-        'is-selected': isSelected,
-      })}
-      // Listen for `onMouseDown` so that the tab is selected when _pressed_
-      // as this makes the UI feel faster. Also listen for `onClick` as a fallback
-      // to enable selecting the tab via other input methods.
-      onClick={selectTab}
-      onMouseDown={selectTab}
-      role="tab"
-      tabIndex="0"
-      title={title}
-      aria-label={title}
-      aria-selected={isSelected.toString()}
-    >
-      {children}
-      {count > 0 && !isWaitingToAnchor && (
-        <span className="selection-tabs__count"> {count}</span>
-      )}
-    </button>
+    <div>
```

Comment:
What is the div needed for?

Reply:
I believe it is intended as a "neutral" layout container for the contained `button`. IIRC, its parent has a `horizontal-rhythm` applied and there were layout funnies with having the `button` elements be direct descendants.